apps/web/src/pages/PoolCandidates/ViewPoolCandidatePage/components/ApplicationInformation/EducationRequirementsDisplay.tsx

```diff
@@ -10,14 +10,119 @@ import { notEmpty } from "@gc-digital-talent/helpers";
 
 import ExperienceTreeItems from "~/components/ExperienceTreeItems/ExperienceTreeItems";
 
+const EducationRequirement_PoolCandidateFragment = graphql(/* GraphQL */ `
+  fragment EducationRequirement_PoolCandidate on PoolCandidate {
+    pool {
+      classification {
+        group
+      }
+    }
+    educationRequirementOption {
+      value
+    }
+    educationRequirementExperiences {
+      id
+      details
+      user {
+        id
+        email
+      }
+      skills {
```

Comment:
I don't quite get what the desired state of the snapshot is, but I am not too sure about this bit. Doesn't this now query for the live experience result attached to `educationRequirementExperiences`? 
Which means if the user updates the experience(s) after application, this component will fetch different states? I think `educationRequirementExperiences` should be stored somehow in the snapshot

Reply:
That is a good point. Though, it seems a little weird to hold application data in the profile snapshot :sweat_smile: 

But yeah, this actually will cause issues with the skills as well :woozy_face:

Reply:
Okay, I think I found a solution. We store the users experiences so if we have the ID then we should be able to get it from those :thinking: Do you think this will work?

[use experiences from snapshot](https://github.com/GCTC-NTGC/gc-digital-talent/pull/11225/commits/00b7d9703a32a6f4f235813dd1632e2aa47334d7)

Reply:
Hm... Maybe? It looks like a good idea to me

Reply:
@tristan-orourke

Reply:
Ohh... I see... It's confusing that you can query for nested details directly on the educationRequirementExperiences, but they're kindof the wrong details :/

I think you have a good solution for now though.

Reply:
As a longer term solution... Make an accessor which actually gets values for educationRequirementExperiences from the snapshot? Essentially, moving this logic to the backend?

Reply:
I'm a little confused, that is what this is doing right now :thinking: 

The education experience is just an array of IDs that point to the experiences so all we need to do is match it with the snapshotted experience (what this does). The reason it happens on the frontend is because if we did that on the backend it would just be duplicating the same data and sending unnecessary bytes does the pipe.

Reply:
In the commit you linked, you deleted a bunch from the graphql request for educationRequirementExperiences and stuck to getting the ids. But, as your old code showed, you _can_ request all the experience details direct from graphql. But, for what's most relevant to the application (the PoolCandidate), those details are kindof wrong. Which is why you're now only getting the ids and making the connection to the snapshot on the frontend. I'm saying, it would probably be better if the query you deleted in [use experiences from snapshot](https://github.com/GCTC-NTGC/gc-digital-talent/pull/11225/commits/00b7d9703a32a6f4f235813dd1632e2aa47334d7) just returned snapshot data in the first place. Moving this logic (connecting experience-id to snapshot-experience) from React to Laravel.

Reply:
Yeah, that is why I am confused. It would be the exact same data though no? So we would be sending it down twice, once in the experiences array and then again in the education experiences. That was the main reason I deleted it, because the data was identical, we just needed the ID to connect the two. Maybe I am missing something but it seems the following is being proposed:

```json
{
  "experiences": [
     {
       "id": 1,
        "title": "Education experience",
     }
  ],
  "educationRequirementExperiences": [
    {
      "id": 1,
      "title": "Education experience"
    }
  ]
}
```

Unless I am missing something :sweat_smile:

Reply:
I feel like creating an accessor and schema addition would be more worth it if we needed to reuse it elsewhere